### PR TITLE
feat: add basic page templates

### PR DESCRIPTION
**Pull request by melaniebmn**
# Summary | Résumé

Adding the following new templates:

- Basic page template (EN)
- Basic page template (FR)
- Basic page table of contents template (EN)
- Basic page table of contents template (FR)

[Zenhub ticket](https://app.zenhub.com/workspaces/design-system-6100624a19f4cf000e46e458/issues/gh/cds-snc/design-gc-conception/1047)

Note: Holding off from merging this PR until design is ready to release.

### CODE REVIEW

File: templates/english/basic-page-template-table-of-contents.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
+<!-- TO DO: Remove all comments before deploying your code to production. -->
```
Comment (daine) [inline templates/english/basic-page-template-table-of-contents.html:1]: Could we change all instances of `TO DO` to `TODO`? I believe the standard usage is the first one, and I know IDEs also grab `TODO` and create a nice list out of it
```suggestion
<!-- TODO: Remove all comments before deploying your code to production. -->
```

**Review by daine — CHANGES_REQUESTED** [review]
While we're waiting for design, just have a small text change request for `TODO`

File: templates/english/basic-page-template-table-of-contents.html (hunk 1 above)
Comment (melaniebmn) [reply templates/english/basic-page-template-table-of-contents.html]: All updated @daine :)

**Review by daine — APPROVED** [review]
LGTM! 

Don't merge; will coordinate with design once the figma design is ready [Zenhub ticket](https://app.zenhub.com/workspaces/design-system-6100624a19f4cf000e46e458/issues/gh/cds-snc/design-gc-conception/1132)

File: templates/english/basic-page-template-table-of-contents.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    />
+
+    <!-- TODO: Insert a concise, descriptive title that summarizes your page's content. -->
+    <title>Basic Page Template (EN)</title>
+
+    <!---------- GC Design System Utility ---------->
+    <!-- TODO: Update the version number to receive future updates of the utility framework. All notable changes will be documented in the changelog: https://github.com/cds-snc/gcds-utility/blob/main/CHANGELOG.md -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-utility@1.3.0/dist/gcds-utility.min.css"
```
Comment (daine) [inline templates/english/basic-page-template-table-of-contents.html:21]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-utility@1.4.0/dist/gcds-utility.min.css"
```

File: templates/english/basic-page-template-table-of-contents.html [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    />
+
+    <!---------- GC Design System Components ---------->
+    <!-- TODO: Update the version number to receive future updates of GC Design System Components. All notable changes will be documented in the changelog: https://github.com/cds-snc/gcds-components/blob/main/CHANGELOG.md -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.25.0/dist/gcds/gcds.css"
```
Comment (daine) [inline templates/english/basic-page-template-table-of-contents.html:28]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.css"
```

File: templates/english/basic-page-template-table-of-contents.html [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    />
+    <script
+      type="module"
+      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.25.0/dist/gcds/gcds.esm.js"
```
Comment (daine) [inline templates/english/basic-page-template-table-of-contents.html:32]:
```suggestion
      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.esm.js"
```

File: templates/english/basic-page-template-table-of-contents.html [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+    ></script>
+    <script
+      nomodule
+      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.25.0/dist/gcds/gcds.js"
```
Comment (daine) [inline templates/english/basic-page-template-table-of-contents.html:36]:
```suggestion
      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.js"
```

File: templates/english/basic-page-template.html [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+    />
+
+    <!-- TODO: Insert a concise, descriptive title that summarizes your page's content. -->
+    <title>Basic Page Template (EN)</title>
+
+    <!---------- GC Design System Utility ---------->
+    <!-- TODO: Update the version number to receive future updates of the utility framework. All notable changes will be documented in the changelog: https://github.com/cds-snc/gcds-utility/blob/main/CHANGELOG.md -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-utility@1.3.0/dist/gcds-utility.min.css"
```
Comment (daine) [inline templates/english/basic-page-template.html:21]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-utility@1.4.0/dist/gcds-utility.min.css"
```

File: templates/english/basic-page-template.html [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+    />
+
+    <!---------- GC Design System Components ---------->
+    <!-- TODO: Update the version number to receive future updates of GC Design System Components. All notable changes will be documented in the changelog: https://github.com/cds-snc/gcds-components/blob/main/CHANGELOG.md -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.25.0/dist/gcds/gcds.css"
```
Comment (daine) [inline templates/english/basic-page-template.html:28]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.css"
```

File: templates/english/basic-page-template.html [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+    />
+    <script
+      type="module"
+      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.25.0/dist/gcds/gcds.esm.js"
```
Comment (daine) [inline templates/english/basic-page-template.html:32]:
```suggestion
      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.esm.js"
```

File: templates/english/basic-page-template.html [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+    ></script>
+    <script
+      nomodule
+      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.25.0/dist/gcds/gcds.js"
```
Comment (daine) [inline templates/english/basic-page-template.html:36]:
```suggestion
      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.js"
```

File: templates/french/basic-page-template-table-of-contents.html [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+    />
+
+    <!-- TODO : Insérer un titre concis et descriptif qui résume le contenu de votre page. -->
+    <title>Modèle de page de base (FR)</title>
+
+    <!---------- GC Design System Utility ---------->
+    <!-- TODO : Mettre à jour le numéro de version pour recevoir les futures mises à jour du cadre utilitaire de Système de design GC. Tout changement important sera consigné dans le journal des modifications: https://github.com/cds-snc/gcds-utility/blob/main/CHANGELOG.md -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-utility@1.3.0/dist/gcds-utility.min.css"
```
Comment (daine) [inline templates/french/basic-page-template-table-of-contents.html:21]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-utility@1.4.0/dist/gcds-utility.min.css"
```

File: templates/french/basic-page-template-table-of-contents.html [hunk 11]
```diff
@@ -0,0 +1,157 @@
...
+    />
+
+    <!---------- GC Design System Components ---------->
+    <!-- TODO : Mettre à jour le numéro de version pour recevoir les futures mises à jour des composants de Système de design GC. Tout changement important sera consigné dans le journal des modifications https://github.com/cds-snc/gcds-components/blob/main/CHANGELOG.md -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.25.0/dist/gcds/gcds.css"
```
Comment (daine) [inline templates/french/basic-page-template-table-of-contents.html:28]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.css"
```

File: templates/french/basic-page-template-table-of-contents.html [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+    />
+    <script
+      type="module"
+      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.25.0/dist/gcds/gcds.esm.js"
```
Comment (daine) [inline templates/french/basic-page-template-table-of-contents.html:32]:
```suggestion
      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.esm.js"
```

File: templates/french/basic-page-template-table-of-contents.html [hunk 13]
```diff
@@ -0,0 +1,157 @@
...
+    ></script>
+    <script
+      nomodule
+      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.25.0/dist/gcds/gcds.js"
```
Comment (daine) [inline templates/french/basic-page-template-table-of-contents.html:36]:
```suggestion
      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.js"
```

File: templates/french/basic-page-template.html [hunk 14]
```diff
@@ -0,0 +1,115 @@
...
+    />
+
+    <!--TODO : Insérer un titre concis et descriptif qui résume le contenu de votre page. -->
+    <title>Modèle de page de base (FR)</title>
+
+    <!---------- GC Design System Utility ---------->
+    <!--TODO : Mettre à jour le numéro de version pour recevoir les futures mises à jour du cadre utilitaire de Système de design GC. Tout changement important sera consigné dans le journal des modifications: https://github.com/cds-snc/gcds-utility/blob/main/CHANGELOG.md -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-utility@1.3.0/dist/gcds-utility.min.css"
```
Comment (daine) [inline templates/french/basic-page-template.html:21]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-utility@1.4.0/dist/gcds-utility.min.css"
```

File: templates/french/basic-page-template.html [hunk 15]
```diff
@@ -0,0 +1,115 @@
...
+    />
+
+    <!---------- GC Design System Components ---------->
+    <!--TODO : Mettre à jour le numéro de version pour recevoir les futures mises à jour des composants de Système de design GC. Tout changement important sera consigné dans le journal des modifications https://github.com/cds-snc/gcds-components/blob/main/CHANGELOG.md -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.25.0/dist/gcds/gcds.css"
```
Comment (daine) [inline templates/french/basic-page-template.html:28]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.css"
```

File: templates/french/basic-page-template.html [hunk 16]
```diff
@@ -0,0 +1,115 @@
...
+    />
+    <script
+      type="module"
+      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.25.0/dist/gcds/gcds.esm.js"
```
Comment (daine) [inline templates/french/basic-page-template.html:32]:
```suggestion
      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.esm.js"
```

File: templates/french/basic-page-template.html [hunk 17]
```diff
@@ -0,0 +1,115 @@
...
+    ></script>
+    <script
+      nomodule
+      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.25.0/dist/gcds/gcds.js"
```
Comment (daine) [inline templates/french/basic-page-template.html:36]:
```suggestion
      src="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-components@0.26.2/dist/gcds/gcds.js"
```

**Review by daine — CHANGES_REQUESTED** [review]
We need to update the versions to the newest before we merge this in

File: templates/english/basic-page-template-table-of-contents.html [hunk 18]
```diff
@@ -0,0 +1,157 @@
...
+    <meta
+      name="description"
+      content="Add a description to provide a brief summary of the content."
+    />
+
+    <!-- TODO: Insert a concise, descriptive title that summarizes your page's content. -->
+    <title>Basic Page Template (EN)</title>
+
+    <!---------- GC Design System Utility ---------->
+    <!-- TODO: Replace <version-nubmer> with the latest version number to receive future updates of the GC Design System utility framework. All notable changes will be documented in the changelog: https://github.com/cds-snc/gcds-utility/blob/main/CHANGELOG.md -->
```
Comment (daine) [inline templates/english/basic-page-template-table-of-contents.html:18]:
```suggestion
    <!-- TODO: Replace <version-number> with the latest version number to receive future updates of the GC Design System utility framework. All notable changes will be documented in the changelog: https://github.com/cds-snc/gcds-utility/blob/main/CHANGELOG.md -->
```

File: templates/english/basic-page-template-table-of-contents.html [hunk 19]
```diff
@@ -0,0 +1,157 @@
...
+    <link
+      rel="stylesheet"
+      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-utility@<version-nubmer>/dist/gcds-utility.min.css"
```
Comment (daine) [inline templates/english/basic-page-template-table-of-contents.html:21]:
```suggestion
      href="https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-utility@<version-number>/dist/gcds-utility.min.css"
```

**Review by daine — CHANGES_REQUESTED** [review]
Small request to change nubmer to number 😅

File: templates/english/basic-page-template-table-of-contents.html (hunk 18 above)
Comment (melaniebmn) [reply templates/english/basic-page-template-table-of-contents.html]: Argh oops - should be fixed!

File: templates/english/basic-page-template-table-of-contents.html (hunk 19 above)
Comment (melaniebmn) [reply templates/english/basic-page-template-table-of-contents.html]: Should be fixed now!

**Review by daine — DISMISSED** [review]

**Review by daine — DISMISSED** [review]
LGTM

**Comment by melaniebmn** [comment]
@daine updated the basic page templates with the new spacing classes so I will need (hopefully) one last approval for this PR.  :crossed_fingers:

**Review by ethanWallace — APPROVED** [review]
LGTM!

**Review by daine — APPROVED** [review]

**Pull request merged**